docs/README.md

```diff
@@ -1,16 +1,42 @@
-# Developer-local docs build
+# Building documentation locally
+
+## Building in an existing envoy development environment
+
+If you have an existing envoy development environment, you should have the necessary dependencies
+and requirements and be able to build the documentation directly.
```

Comment:
can someone confirm this is correct?

i dont currently have a "dev env" so just wanted to confirm

Reply:
I think this is true, but I don't have a clean slate to start with. Can you link from "existing Envoy development environment" to https://github.com/envoyproxy/envoy/tree/master/bazel#quick-start-bazel-build-for-developers?

Reply:
yep, will do...